doc/classes/EditorInterface.xml

```diff
@@ -159,9 +159,10 @@
 		<method name="select_file">
 			<return type="void">
 			</return>
-			<argument index="0" name="file" type="String">
+			<argument index="0" name="file_path" type="String">
 			</argument>
 			<description>
+				Select the file in the FileSystem dock using the [code]filepath[/code] provided.
```

Comment:
`Selects`, the convention is that the method is the subject of the verb in docstrings.